### PR TITLE
[ImportExport] Cover Export Source Model by Unit Test

### PR DESCRIPTION
**Pull request by edenduong**
### Description (*)

**Increase Test Coverage**

Cover Export Source Model by Unit Test
- Magento\ImportExport\Model\Source\Export\Format
- Magento\ImportExport\Model\Source\Export\Entity

### Manual testing scenarios (*)

- vendor/bin/phpunit app/code/Magento/ImportExport/Test/Unit/Model/Source/Export/FormatTest.php
- vendor/bin/phpunit app/code/Magento/ImportExport/Test/Unit/Model/Source/Export/EntityTest.php

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: app/code/Magento/ImportExport/Test/Unit/Model/Source/Export/EntityTest.php [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    }
+
+    /**
+     * Test toOptionArray with data provider
+     *
+     * @param array $entities
+     * @param array $expected
+     * @dataProvider toOptionArrayDataProvider
+     */
+    public function testToOptionArray($entities, $expected)
```
Comment (lbajsarowicz) [inline app/code/Magento/ImportExport/Test/Unit/Model/Source/Export/EntityTest.php:52]: You can use type hints there: `array $entities, array $expected`

File: app/code/Magento/ImportExport/Test/Unit/Model/Source/Export/EntityTest.php [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+        $this->assertEquals($expected, $this->model->toOptionArray());
+    }
+
+    /**
+     * Data Provider for test toOptionArray
+     *
+     * @return array
+     */
+    public function toOptionArrayDataProvider()
```
Comment (lbajsarowicz) [inline app/code/Magento/ImportExport/Test/Unit/Model/Source/Export/EntityTest.php:64]: You can use return type there `...(): array`

File: app/code/Magento/ImportExport/Test/Unit/Model/Source/Export/FormatTest.php [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    }
+
+    /**
+     * Test toOptionArray with data provider
+     *
+     * @param array $fileFormats
+     * @param array $expected
+     * @dataProvider toOptionArrayDataProvider
+     */
+    public function testToOptionArray($fileFormats, $expected)
```
Comment (lbajsarowicz) [inline app/code/Magento/ImportExport/Test/Unit/Model/Source/Export/FormatTest.php:52]: Type hints

File: app/code/Magento/ImportExport/Test/Unit/Model/Source/Export/FormatTest.php [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+
+        $this->assertEquals($expected, $this->model->toOptionArray());
+    }
+
+    /**
+     * Data Provider for test toOptionArray
+     *
+     * @return array
+     */
+    public function toOptionArrayDataProvider()
```
Comment (lbajsarowicz) [inline app/code/Magento/ImportExport/Test/Unit/Model/Source/Export/FormatTest.php:64]: Return type

**Review by lbajsarowicz — APPROVED** [review]
Thank you for contribution.
You can add return types and type hints, but it's not critical.

**Comment by magento-engcom-team** [comment]
Hi @lbajsarowicz, thank you for the review.
[ENGCOM-6509](https://jira.corp.magento.com/browse/ENGCOM-6509) has been created to process this Pull Request
:eight_spoked_asterisk: @lbajsarowicz, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Comment by engcom-Delta** [comment]
**Notice:** QA not applicable

**Pull request merged**